### PR TITLE
Add SpeedreaderReleaseStudy to release channel

### PR DESCRIPTION
**Pull request by keur**
We want to do a partial rollout of Speedreader on Release.

### CODE REVIEW

**Comment by kjozwiak** [comment]
Waiting on a dev to review the above to make sure code/formatting looks correct 👍 Once merged into `main`, QA will quickly check to make sure the above is working on `staging` before we get it merged into `production`.

File: seed/seed.json
Comment (iefremov) [inline seed/seed.json:244]: pls check other entries in this file, this has format chromium + brave, e.g. `92.1.28.xxxx`

Comment (iefremov) [reply seed/seed.json:244]: @moritzhaller maybe we could add a check for this field? e.g. second digit should always be 1, as in Brave

Comment (keur) [reply seed/seed.json]: fixed

**Review by iefremov — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 Lets get this merged and get a `production` PR created.

**Comment by keur** [comment]
> LGTM  Lets get this merged and get a `production` PR created.

@kjozwiak prod PR is here https://github.com/brave/brave-variations/pull/92

**Pull request merged**
